### ISSUE TITLE
Offline bundle map not working

### ISSUE THREAD

**Issue by ghost** [TrekDev/Xamarin.Android.Skobbler#5]
Hi again TrekDev,

I tried to integrate a pre-bundle map in my app (the files were given to me by Skobbler, so they should be ok) without success. I followed the instructions on http://sdkblog.skobbler.com/creating-an-app-with-a-pre-bundled-map/ and tried to apply what I believe is the translation you did:

```
    private void InitializeLibrary()
    {
        var app = Application as DemoApplication;

        if (app == null) return;

        var initMapSettings = new SKMapsInitSettings();
        initMapSettings.SetMapResourcesPaths(app.MapResourcesDirPath, new SKMapViewStyle(app.MapResourcesDirPath + "daystyle/", "daystyle.json"));
        initMapSettings.PreinstalledMapsPath = _mapResourcesDirPath +"/PreinstalledMaps";
        initMapSettings.ConnectivityMode = SKMaps.ConnectivityModeOffline;
        SKAdvisorSettings advisorSettings = initMapSettings.AdvisorSettings;
        advisorSettings.Language = "en";
        advisorSettings.AdvisorVoice = "en";
        initMapSettings.AdvisorSettings = advisorSettings;

        SKVersioningManager.Instance.SetMapUpdateListener(this);
        SKMaps.Instance.InitializeSKMaps(this, initMapSettings, ApiKey);
    }
```

With this code it fails to calculate route ("route calculation failed") and don't render the map.

Thank you for your time,

RA

EDIT: this is for SDK 2.3.

**Comment by jeremy-ellis-tech** [comment]
Hi RA,

Sorry it's taken me a while to get back to you. Unfortunately I haven't been able to try this since I'm without a suitable development environment at the moment.

How are you loading the files on the device? My first guess would be permission problems.
If it is a bug I probably won't be able to provide a nice fix since I've moved ahead with v2.4.0 and
I was fixing something similar last weekend in v2.4.0. My suggestion would be to upgrade to v2.4.0 (I think it provides some of the pedestrian navigation you were looking for in a previous issue) and see if the same problem exists there. If it does I can fix it as soon as I get a chance this weekend.

Thanks,

-TD

**Comment by ghost** [comment]
Hello TrekDev,

I'll try to see if I can fix it with the permissions later. This weekend I'll move to 2.4 and I'll keep you posted of any issue that comes across.

Thank you again for your work!

RA
